src/explore-education-statistics-common/src/modules/table-tool/components/FilterHierarchy.tsx

```diff
@@ -51,33 +80,148 @@ function FilterHierarchy({
+          }
+          showClose
+          closeText="Close"
         >
-          {Object.keys(filterHierarchy[0].hierarchy).map(optionId => {
+          {filterLabels.map((filterLabel, index) => {
+            const isFirst = index === 0;
+            const isLast = index + 1 === filterLabels.length;
+
             return (
-              <FilterHierarchyOptions
-                key={optionId}
-                optionId={optionId}
-                optionLabelsMap={optionLabelsMap}
-                filterHierarchy={filterHierarchy}
-                name={name}
-                disabled={disabled}
-                tiersTotal={tiersTotal}
-                selectedValues={selectedValues}
-                level={0}
-              />
+              <div
+                key={filterLabel}
+                className={classNames(styles.tier, {
+                  [styles.subTier]: !isFirst,
+                })}
+              >
+                <h3 className="govuk-heading-s">
+                  {filterLabel} (tier {index + 1})
+                </h3>
+                {isFirst && !isLast && (
+                  <>
+                    <p>This is a top level category.</p>
+                    <p>
+                      Selecting a tier 1 option provides a total value for all
+                      sub categories associated to this area
+                    </p>
+                  </>
+                )}
+                {!isFirst && !isLast && (
+                  <>
+                    <p>This is a sub category of tier {index}.</p>
+                    <p>
+                      Selected a tier {index + 1} option provides a total value
+                      for all {filterLabels[index + 1].toLowerCase()} options
+                      associated to this area
+                    </p>
+                  </>
+                )}
+                {isLast && (
+                  <>
+                    <p>These are the individual {filterLabel.toLowerCase()}</p>
+                    <p>
+                      If you're not sure about a specific{' '}
+                      {filterLabel.toLowerCase()} options, then you can also use
+                      the browse feature to find via related parent tiers
+                    </p>
+                  </>
+                )}
+              </div>
             );
           })}
-        </FormFieldset>
-      </div>
+        </Modal>
+        {rootOptionTrees.map(optionTree => {
+          return (
+            <FilterHierarchyOptions
+              key={optionTree.value}
+              optionTree={optionTree}
+              name={name}
+              disabled={disabled}
+              selectedValues={selectedValues}
+              level={0}
+              toggleOptions={toggleOptions}
+              expandedOptionsList={expandedOptionsList}
+            />
+          );
+        })}
+      </FormFieldset>
     </DetailsMenu>
   );
 }
 
+function sortAlphabeticalTotalsFirst(a: string, b: string) {
```

Comment:
Is this a duplicate of https://github.com/dfe-analytical-services/explore-education-statistics/blob/dev/src/explore-education-statistics-common/src/utils/sortAlphabeticalTotalsFirst.ts ?

Reply:
Yes it was, I thought the original was removed in previous refactor, good spot!